src/Resources/config/cors.yml

```diff
@@ -0,0 +1,3 @@
+nelmio_cors:
+    paths:
+        '/': # Nonsense configuration as the nelmio compiler pass will abort if not present
```

Comment:
I have created a PR to fix this: https://github.com/nelmio/NelmioCorsBundle/pull/73